### ISSUE TITLE
Stories with @issue metadata duplicated in Serenity Report Overall Test Results page

### ISSUE THREAD

**Issue by teggr** [serenity-bdd/serenity-core#90]
Using this serenity version:

```
<serenity.version>1.0.59</serenity.version>
```

I have a jbehave story stored under the following structure:

```
+ stories
   + name_of_my_epic
      + my_story.story
```

The story contains an @issue annotation

```
My story

Meta
@issue PRO-5760

Narrative:
...
```

When I run the tests and generate the report, I get duplicated entries in the list of stories on the overall test results page. The entries appear to be different in the title. One has the issue as the parent, the other my epic.

If I click the first link, then the story title is shown as

Story: PRO-5760/My story

If I click the second link, then the story title is shown as 

Story: Name of my epic/My Story

I would expect only a single entry in this list.

If I go to the story directly through the stories tab, then the output is as expected and I only see one entry

**Comment by wakaleo** [comment]
Hi, I'm having trouble reproducing this with the current version, could you create a sample project that reproduces the issue?

**Comment by teggr** [comment]
Hi John,

I've created an example project on github: [serenity-example-project](https://github.com/teggr/serenity-example-project).

Run: mvn clean verify

I see the following behaviour in the overall test results page

### RUN WITH @issue on separate line

Meta:
@issue IHP-1235

Stories | 
------------ | 
Issue In Filename |
Matching Filename And Title |
Ihp 1234 Issue In Filename |	
Matching Filename And Title |	

The scenarios have been assigned the jira issues

Tests |
-------- |
Account has blah blah (IHP-1235) |
Account has whatever (IHP-1234) |

### RUN WITH @issue on same line as Meta

Meta: @issue IHP-1235

Stories |
--------- |
hp 1234 Issue In Filename |
Issue In Filename	|
Matching Filename And Title	|

The scenarios have been assigned the jira issues

Tests |
------ |
Account has blah blah (IHP-1235) |
Account has whatever (IHP-1234) |

**Comment by YamStranger** [comment]
Hello @teggr 

Thanks for this issue!! 

I tested behaving with your project using latest version of serenity 
```
	<properties>
		<serenity.version>1.1.27</serenity.version>
		<serenity.maven.version>1.1.26</serenity.maven.version>
		<serenity.jira.version>1.1.2-rc.1</serenity.jira.version>
		<serenity.jbehave.version>1.7.0</serenity.jbehave.version>
	</properties>
```

when `@issue` tag provided on separate line against `Meta:`
report looks like: 
![stories_separ_line](https://cloud.githubusercontent.com/assets/1667699/13451757/b973a2aa-e04a-11e5-80e6-27cea0466771.png)
![tests_separ_line](https://cloud.githubusercontent.com/assets/1667699/13451760/b9777b64-e04a-11e5-8b96-41fa82fcecfc.png)

If `@issue` tag provided on same line with `Meta:`

![story_same_line](https://cloud.githubusercontent.com/assets/1667699/13451758/b974b83e-e04a-11e5-803d-eb326d941a4a.png)
![tests_same_line](https://cloud.githubusercontent.com/assets/1667699/13451759/b974c02c-e04a-11e5-8069-8dfee7a7cced.png)

Your but was fixed, when we move jbehave from 3.0.5 to 4.0.5 - no duplication any more. But It seems that we have additional bug - if some `@tag` tag prodvided on same line as Meta - it can not be found. I created issue for these in jbehave module: serenity-bdd/serenity-jbehave/issues/56

Will close this issue
